src/cluster/manager_create.go

```diff
@@ -256,6 +256,8 @@ func (m *Manager) createCluster(
 
 		err = exec.Get(ctx, nil)
 		if err != nil {
+			_ = cluster.SetStatus(pkgCluster.Error, "failed to run setup jobs")
```

Comment:
Shouldn't this be rather 
```suggestion
			_ = cluster.SetStatus(pkgCluster.Error, emperror.Wrap(err, "running setup jobs failed").Error())
```